lib/rails_multitenant/global_context_registry.rb

```diff
@@ -85,6 +85,27 @@ def replace_registry(registry)
       self.globals = registry
     end
 
+    # Run a block of code that disregards scoping during read queries
+    def with_admin_registry
+      puts globals
+      prior_globals = new_registry(globals.merge(admin_registry_enabled: true))
```

Comment:
Can you use `with_isolated_registry` or `with_merged_registry` instead?

Reply:
Yes, `with_merged_registry` works.